src/plugins/enzymes/src/FindEnzymesDialog.cpp

```diff
@@ -898,6 +953,13 @@ QString EnzymeTreeItem::getEnzymeInfo() const {
     QString result;
     result += QString("<a href=\"http://rebase.neb.com/rebase/enz/%1.html\">%1</a>")
                   .arg(text(Column::Id));
+    if (enzymesNumber != INCORRECT_ENZYMES_NUMBER) {
+        if (enzymesNumber > MAXIMUM_ENZYMES_NUMBER) {
+            result += tr(" (>10000 sites)");
```

Comment:
The constant should be a part of the text: we can change it later.

Reply:
OK